### DISCUSSION TITLE
milvus直接查询指定表，找不到数据

### DISCUSSION THREAD

**Discussion by gaoyuan98** [milvus-io/milvus#25968] [Q&A and General discussion]
我直接加载并查询已经在库中的表的向量数据查不到
![image](https://github.com/milvus-io/milvus/assets/44916277/5054c842-4de5-47b6-94e9-edcddc62552e)
于是我新建了一个从连接到插入再到查询，是可以找到的，我现在找不到问题在哪里。希望可以指点下
这个是从插入在到查询是可以的，新建文件直接查询全找不到
![image](https://github.com/milvus-io/milvus/assets/44916277/f58f781d-c8ce-47fe-b1cb-6b93f2dcbe8e)

**Comment by yhmo** [comment]
建表的时候是否设置了“collection.ttl”， 类似这样：
collection = Collection(name=name, data=None, schema=schema, properties={"collection.ttl.seconds": 15})

"collection.ttl.seconds": 15， 这样会自动清楚表里存在时间超过15秒的数据。pymilvus的example.py里是这么建表的，如果你从那里copy建表语句，那就会有这个问题。

**Comment by yhmo** [comment]
另外：milvus社区里鼓励使用英文，尽量使用英文提问和对话，谢谢

**Comment by gaoyuan98** [comment]
i not set collection.ttl, right is my create table method，but left no query table data

![image](https://github.com/milvus-io/milvus/assets/44916277/f4ee6086-f2d2-4d47-9102-a18857e968ef)

**Comment by gaoyuan98** [comment]
I think I have a problem with this code, can you help take a look
![image](https://github.com/milvus-io/milvus/assets/44916277/05922d73-14ff-4406-a8d7-5b14b055aa91)

**Comment by gaoyuan98** [comment]
After testing, I gave the connections.connect() function to pass parameters and the query failed, and the query succeeded without passing parameters, I didn't understand a little

![image](https://github.com/milvus-io/milvus/assets/44916277/8fd38f20-a6be-4ae7-a112-9f9dfff86de1)

**Comment by gaoyuan98** [comment]
After testing, remove the attribute db_name, if the db_name is wrong, shouldn't you report an error or give a hint

And I also created db_name attribute, but I actually removed it I don't know why



![image](https://github.com/milvus-io/milvus/assets/44916277/09cf3b8f-b4cc-4442-aaac-c6a77c773a11)

**Reply by xiaofan-luan** [reply]
@czs007 
could you help on the db verification?

if user access on a not existed db,  we should report error?
